components/PriorityBillsCard/PriorityBillsCard.tsx

```diff
@@ -19,13 +19,14 @@ export const PriorityBillsCard = (props: {
   editBtn: boolean
 }) => {
   const items = props.bills?.map((bill, index) => {
-    let style = Styles.billSlot
-    if (bill.billNumber === props.selectedBillId) {
-      style = Styles.billSelected
-    }
+    const isSelectedBill = bill.billNumber === props.selectedBillId
+
     return bill.stance ? (
       <ListItem
-        className={style}
+        className={clsx(
+          "bg-secondary text-white" && !isSelectedBill,
```

Comment:
This styling is not continuing through. There are a few things in the way, e,g. we aren't using classNames that we pass to the ListItem component. Ultimately it would be good to refactor this making better use of the react-bootstrap tools, but since this is an inactive component it doesn't make sense to do that now.